environments/Test-Laptop.json

```diff
@@ -41,6 +41,9 @@
         "hdp_utils": "http://public-repo-1.hortonworks.com/HDP-UTILS-1.1.0.20/repos/ubuntu12"
       },
       "hadoop" : {
+        "environment" : {
```

Comment:
@pu239ppy it is a known that there is a environment name in ``chef`` environment file. This attribute is for users to identify the workload profile which can be different from the environment name and if users wish they can set this attribute to the same value as the ``chef`` environment name.